test.sh

```diff
@@ -23,9 +23,9 @@ rm -R p || true
 echo
 echo "==> Starting tests with value log mmapped..."
 sleep 5
-go test -v --vlog_mmap=true -race ./...
+go test -v --vlog_mmap=true -race -timeout 900s ./...
```

Comment:
@ashish-goswami why do we need to increase the timeout? Is there a test that doesn't complete in 10 minutes (default timeout) ?

Reply:
Once my build failed with message `tests ran too long` and hence changed it. I have removed the commit now. It was just for debugging.